loading-button-android/src/main/java/br/com/simplepass/loadingbutton/customViews/ProgressButton.kt

```diff
@@ -82,12 +85,20 @@ internal fun ProgressButton.init(attrs: AttributeSet? = null, defStyleAttr: Int
         getContext().obtainStyledAttributes(this, attrsArray, defStyleAttr, 0)
     }
 
-    drawable = parseGradientDrawable(
-        typedArrayBg?.getDrawable(0)
+    val tempDrawable = typedArrayBg?.getDrawable(0)
```

Comment:
Thanks for pointing tempDrawable it is as you say. 
parseGradientDrawable was removed because at this point we could have other Drawables that cannot be cast straight forward to Gradient and we use that once we need to start animation

Reply:
I've tested it and unfortunately now I remember that we need to use temp variable because if we don't then cache mechanism will infect corners for all buttons using same cache. This will create issue where all buttons using same cache will have after animation corner radius